src/dispdrivers/vgacontroller.cpp

```diff
@@ -151,33 +151,95 @@ void IRAM_ATTR VGAController::VSyncInterrupt(void * arg)
 }
 
 
+std::function<uint8_t(RGB888 const &)> VGAController::getPixelLamda(PaintMode mode)
+{
```

Comment:
to maintain compatibility with existing fab-gl code that used the `NOT` paint option, we may wish to change these lambda-factories to accept the paintOptions object and return PaintMode::Inverse style functions

if, on the other hand, we consider that compatibility to not be important, that this codebase essentially is for use of agon-vdp only, then we don't need to continue to support `NOT`